CONTRIBUTING.md

```diff
@@ -59,3 +61,23 @@ To create a new rule:
     ```
 
 10. Add the rule tests under `test/rules/`. Prefer to split test examples to a correct/incorrect groups.
+
+## Run the plugin in IDE
+
+The plugin works in multiple IDEs and is implemented as analysis server plugin. To work on and test IDE/analysis server integration features and issues you
+will need to load the plugin into analysis server.
+
+To set this up:
+
+1. Assume you have cloned the repository into `<absolute-path>` directory.
```

Comment:
Suggestion: Clone the repository into `<absolute-path>` directory.

I think we can simplify it (if the user had already cloned the repo, they will just skip this step).